.circleci/config.yml

```diff
@@ -624,7 +624,7 @@ jobs:
               steps:
                   - run:
                       command: |
-                          E2E_TRIGGER=$(./scripts/ci/ci-e2e-job-trigger-checker.sh || echo -n "retval $?")
+                          E2E_TRIGGER=$(./scripts/ci-e2e-job-trigger-checker.sh || echo -n "retval $?")
```

Comment:
why was this path incorrect? did we move something or was it a typo from the beginning?

Reply:
I guess is that it was copied from another repo to the wrong folder in our.